Gui Rebuild

SandervDoorn opened the pull request:
First itteration of the new GUI

SandervDoorn committed:
Relocated files and cleaned up connectionmanager

SandervDoorn committed:
Created new main application that supports dynamically creation of Shutter objects
Shutter objects fully self managing

SandervDoorn committed:
Application now dynamically creates shutters and checks for new devices every 10 seconds

SandervDoorn committed:
Moved old GUI to folder "old"
GUI now supports removing and adding devices automatically
Updated protocolhandeler.c

SandervDoorn committed:
GUI updates only new devices and doesnt freak out

SandervDoorn committed:
Added details window popup, no content

Triplays reviewed (approved):
helemaal topie jongen

SandervDoorn committed:
Updated protocol now has error handeling
Added get_settings to improve response times
Added parser to Zonnescherm to support error handeling

SandervDoorn committed:
Main screen now shows shutters in a grid

SandervDoorn committed:
Added graph to details window
